### PR TITLE
[tflchef] Support SegmentSum operator

### PR DESCRIPTION
**Pull request by d-krylov**
Support SegmentSum operator in tflchef

ONE-DCO-1.0-Signed-off-by: Denis Krylov <krylovdenism@gmail.com>

### CODE REVIEW

File: compiler/tflchef/core/src/OpChef.def [hunk 1]
```diff
@@ -71,6 +71,8 @@ OP_CHEF(ReverseSequence, ReverseSequenceChefFactory)
 OP_CHEF(Round, RoundChefFactory)
 OP_CHEF(Rsqrt, RsqrtChefFactory)
 OP_CHEF(ScatterNd, ScatterNdChefFactory)
+OP_CHEF(SegmentSum,SegmentSumChefFactory)
+OP_CHEF(Shape, ShapeChefFactory)
```
Comment (mhs4670go) [inline compiler/tflchef/core/src/OpChef.def:75]: This might be added mistakenly.

File: compiler/tflchef/proto/tflchef.proto [hunk 2]
```diff
@@ -492,7 +496,7 @@ message Operation {
   optional Pool2DOptions l2pool2d_options = 203;
   optional ReducerOptions all_options = 204;
   optional ReducerOptions reduce_min_options = 205;
-
+  optional SegmentSumOptions segment_sum_options = 206;
```
Comment (binarman) [inline compiler/tflchef/proto/tflchef.proto:499]: removed empty line

File: compiler/tflchef/core/src/OpChef.def (hunk 1 above)
Comment (binarman) [reply compiler/tflchef/core/src/OpChef.def:75]: Indeed!
```suggestion
```

File: compiler/tflchef/proto/tflchef.proto (hunk 2 above)
Comment (binarman) [reply compiler/tflchef/proto/tflchef.proto:499]:
```suggestion
  optional SegmentSumOptions segment_sum_options = 206;

```

**Pull request merged**